wasm-test/src/lib.rs

```diff
@@ -6,9 +6,9 @@ use holochain_wasm_utils::*;
 use hdk::globals::g_mem_stack;
 
 #[no_mangle]
-pub extern "C" fn check_global_dispatch(encoded_allocation_of_input : i32) -> i32 {
+pub extern "C" fn check_global_dispatch(encoded_allocation_of_input : u32) -> u32 {
   unsafe {
```

Comment:
Why do you need this unsafe here?

Reply:
because `g_mem_stack` is a static mutable, which is unsafe to use.

Reply:
Why did you merge? This wasn't done! Why are you being disrespectful?

Reply:
I merged because @zippy and @lucksus approved and I did not see your disapproval.
I did not see it because your comment is not related to the changes in this PR.